app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -58,6 +58,9 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
   lazy val reportAProblemNonJSUrl: String =
```

Comment:
Can you delete this val and the "ajax" one as they are not used (and deprecated I think)

Reply:
Done